docs/add-ons/kubernetes-dashboard.md

```diff
@@ -0,0 +1,42 @@
+# Kubernetes Dashboard
+
+[Kubernetes Dashboard](https://github.com/kubernetes/dashboard) is a general purpose, web-based UI for Kubernetes clusters. It allows users to manage applications running in the cluster and troubleshoot them, as well as manage the cluster itself.
+
+## Usage
+
+The following will deploy the Kubernetes Dashboard into an EKS Cluster.
+
+```hcl-terraform
+enable_kubernetes_dashboard = true
+```
+
+Enable Kubernetes Dashboard with custom `values.yaml`
+
+```hcl-terraform
+  enable_kubernetes_dashboard = true
+
+  # Optional Map value
+  kubernetes_dashboard_helm_config = {
+    name       = "kubernetes-dashboard" # (Required) Release name.
+    repository = "https://kubernetes.github.io/dashboard/" # (Optional) Repository URL where to locate the requested chart.
+    chart      = "kubernetes-dashboard" # (Required) Chart name to be installed.
+    version    = "5.2.0"
+    namespace  = "kube-system"
```

Comment:
Should we create a new namespace instead of using kube-system?

Reply:
Our [docs](https://docs.aws.amazon.com/eks/latest/userguide/dashboard-tutorial.html) refer to `kube-system`. Customers can override through `helm_config` we will honor that.